app/views/prototype2/allocations.html

```diff
@@ -64,8 +65,9 @@ <h2 class="govuk-tabs__title">
             <td aria-label="Arrival date" class="govuk-table__cell ">{{prisoner.arrival_date}}</td>
             <td aria-label="Release date" class="govuk-table__cell ">{{prisoner.release_date}}</td>
             <td aria-label="Tier" class="govuk-table__cell ">{{prisoner.tier}}</td>
-            <td aria-label="Status change" class="govuk-table__cell ">{{prisoner.status_change}}</td>
-            <td aria-label="Action" class="govuk-table__cell "><a href="/prototype2/prisoner/{{prisoner.id}}">Details</a></td>
+            <td aria-label="Tier" class="govuk-table__cell">{{prisoner.allocation_date}}</td>
+            <td aria-label="POM" class="govuk-table__cell">{{data.poms[prisoner.pomIndex] | prisonerName}}</td>
```

Comment:
Did you want the POM name to be in the same format as prisoners (surname, firstname)? Also there seems to be a lot of information squeezed into the table.

Reply:
Yes, that filter needs to rename that method. Yep a lot of data

Reply:
Not sure what you mean. I meant that you could use the 'fullName' filter instead of the 'prisonerName' filter to keep POM names in the [firstname surname] format (which is used on the reallocate page).